tests/coverage/coverage.cpp

```diff
@@ -80,7 +78,7 @@ TEST(Coverage, importWithNonHrefXlink)
     libcellml::Parser parser;
     parser.parseModel(e);
     EXPECT_EQ(size_t(0), parser.errorCount());
-//    EXPECT_EQ("bob", parser.error(0)->description());
+    //    EXPECT_EQ("bob", parser.error(0)->description());
```

Comment:
Is that something that you actually want to merge in? I am definitely not in favour of merging code that has been commented out.

Reply:
Yes ... and no, I left it in to remind myself of the error that gets generated when testing the attribute.  I'll take it out.